README.md

```diff
@@ -24,11 +24,17 @@ Looking for the webpack 1 loader? Check out the [archive/webpack-1 branch](https
 <h2 align="center">Install</h2>
 
 ```bash
-npm install sass-loader node-sass webpack --save-dev
+npm install sass-loader sass webpack --save-dev
```

Comment:
Why there is dart sass installation if node-sass is the default implementation?

> `"sass-loader" // compiles Sass to CSS, using Node Sass by default` later in an example in this file.

Reply:
Oops, I missed this. Changed back to `node-sass`.